dtbase/models/arima/run_pipeline.py

```diff
@@ -41,6 +41,8 @@ def run_pipeline(session: Optional[Session] = None) -> None:
 
     # fetch training data from the database
     sensor_data = get_training_data()
+    if not sensor_data:
```

Comment:
These lines weren't supposed to be in this commit. They are harmless, but don't really relate to anything else here, sorry.